src/components/ButtonPanel.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Buttton from './Button';
+
+const ButtonPanel = () => (
+  <>
+    <div className="group1">
+      <Buttton name="AC" />
+      <Buttton name="+/-" />
+      <Buttton name="%" />
+      <Buttton name="/" />
+    </div>
+    <div className="group2">
+      <Buttton name="7" />
+      <Buttton name="8" />
+      <Buttton name="9" />
+      <Buttton name="X" />
+    </div>
+    <div className="group3">
+      <Buttton name="4" />
+      <Buttton name="5" />
+      <Buttton name="6" />
+      <Buttton name="-" />
+    </div>
+    <div className="group4">
+      <Buttton name="1" />
+      <Buttton name="2" />
+      <Buttton name="3" />
+      <Buttton name="+" />
+    </div>
+    <div className="group5">
+      <Buttton name="0" />
+      <Buttton name="." />
+      <Buttton name="=" />
+    </div>
```

Comment:
### [Optional]
Your code appears to be WET-(Write Every Time), Microverse encourage DRY-(Don't Repeat Yourself) code, In other words; you repeated the `Button` component multiple times.

You can use [objects.key](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)  and `map` to loop through the objects. `objects.keys` for the outer loop and `map` for the inner loop.

Example; 
```
const GroupButtons = {
  Group 1: ['AC', '+/-', '%', '÷'],
  Group 2: ['7', '8', '9', 'X'],
   ............
}
```